app-localize-behavior.html

```diff
@@ -194,6 +195,10 @@
     }
   },
 
+  ready() {
+    this.fire('app-localize-behavior-created', this);
```

Comment:
I'm tempted to say _your_ implementation of the element should fire this event, not the behaviour itself. I don't think any other element fires an event like this, and `ready` isn't a great time for you to do anything with this behaviour